### PR TITLE
Remove jQuery and test without it

### PR DESCRIPTION
**Pull request by cibernox**

### CODE REVIEW

File: tests/unit/components/async-button-test.js [hunk 1]
```diff
@@ -1,90 +0,0 @@
-import {
-  moduleForComponent,
```
Comment (cibernox) [inline tests/unit/components/async-button-test.js]: Converted to integration tests. Unit tests don't work without jQuery, but regardless of I find them rather useless.

**Comment by cibernox** [comment]
@bcardarella I believe that you have to remove the circle-ci hook yourself.

File: tests/unit/components/async-button-test.js (hunk 1 above)
Comment (rwjblue) [inline tests/unit/components/async-button-test.js]: 👍

**Comment by bcardarella** [comment]
Done!

**Pull request merged**